### PR TITLE
Network Switch in Review Stat

### PR DESCRIPTION
**Pull request by davidkoller318**
Hi, @chillvybz 
Fixed issue https://github.com/polkaswitch/polkaswitch-ui/issues/35

Please review it. 
Thank you

### CODE REVIEW

File: src/client/js/components/partials/swap/SwapNetworkToggle.jsx [hunk 1]
```diff
@@ -73,7 +94,9 @@ export default class SwapNetworkToggle extends Component {
 
           <div className="level-right">
             <div className="level-item">
-              <div className={classnames("dropdown is-right is-hoverable")}>
+              <div className={classnames("dropdown is-right ", {
+                  "is-hoverable": this.state.hoverable
```
Comment (chillvybz) [inline src/client/js/components/partials/swap/SwapNetworkToggle.jsx:98]: what does this class do?

Comment (chillvybz) [reply src/client/js/components/partials/swap/SwapNetworkToggle.jsx:98]: hm this won't disable the drop-down. We need to disable it (from clicks etc)

Comment (davidkoller318) [reply src/client/js/components/partials/swap/SwapNetworkToggle.jsx:98]: Yes. It's for not allowing network switches in the Review State.

Comment (davidkoller318) [reply src/client/js/components/partials/swap/SwapNetworkToggle.jsx:98]: hm this won't disable the drop-down. We need to disable it 
currently, user can't switch the network status(from clicks etc) in the Review page.

Comment (davidkoller318) [reply src/client/js/components/partials/swap/SwapNetworkToggle.jsx:98]: Screen Shot 2021-08-16 at 10 17 13 AM

**Pull request merged**